sphinx-docs/usermanual/userman_admin.rst

```diff
@@ -204,30 +218,101 @@ Downloading Individual Videos
 #. View subtopics by clicking on the '+' symbol to the left of a subject of your choice. You can cose them by clicking on the '-' symbol.
 #. Mark the content you wish to download by clicking the checkbox to the left of the content name. 
 #. Click the green "Download" button in the top left box of the page. The button should also show you the total number of videos you have selected to download, as well as the total size of the content.
-#. Once the download is completed, video content will be ready for students to watch!
+#. Once the download is completed, video content will be ready for Learners to watch!
 
 Downloading Videos in Bulk
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If you want to download videos in bulk, your best option is to download the KA Lite videos through the `BitTorrent Sync`_ client. This will be a much faster process than using the KA Lite app to download all of the videos.
+
+We have made the full set of KA videos (in the format needed by KA Lite) available via `BitTorrent Sync`_ (btsync). Note that this is different from BitTorrent; btsync allows us to add new videos or fix problems without issuing a whole new torrent file, and then having seeders split between the old and new torrent files. Here are the steps to set this up:
+
+#. Download and install `BitTorrent Sync`_.
+#. Run btsync. On some platforms, this will bring up a graphical interface. On Linux, you will need to load http://127.0.0.1:8888/ to get the interface
+#. Click the "Enter a key or link" button, and put in **https://link.getsync.com/#f=ka­lite­compressed&sz=25E9&q=­&s=TOQVB3LLGWCKEQ6NOCFBSEVWA74PRA6I&q=­&i=C4M3QMAVXE7RVXR7B3526TBD5V3KYV 5V6&q=­&p=CCGAGLIJGFQFP2X2Z4QWQ3MLPJHTVV3A** 
+#. Select the "content" folder inside your KA Lite installation as the "location" (unless you want the videos to be located elsewhere).
+#. Allow the videos to sync in there from your peers! It may take a while for now, as we don't yet have many seeders on it. On that note -- please help seed by keeping it running even after you've got all the videos, if you have bandwidth to spare! This will make it easier for others to download the content as well.
+
+These are resized videos. All in all, this will take around 23 GB of space. 
+
+If you chose to download them to somewhere other than the content folder inside the ka-lite folder as recommended above, you need to tell KA Lite where to find them. If this is the case, follow the steps below:
```

Comment:
It might be good to emphasize this point. Is there some sort of warning or alert directive you can use?